gemrb/core/System/Logger/Stdio.cpp

```diff
@@ -107,18 +102,12 @@ void StdioLogger::LogInternal(LogMessage&& msg)
 	print("\n");
 }
 
-void StdioLogger::destroy()
-{
-	textcolor(DEFAULT);
```

Comment:
Is this handled somewhere else now? I didn't see it. It's was added to ensure we don't leave the vterm in the wrong color if we terminate or stop (gdb) at an inopportune time.

Reply:
ah, good point, I'll add it to the destructor then